.github/workflows/integration_tests.yml

```diff
@@ -393,6 +393,139 @@ jobs:
             exit 1
           fi
 
+  test_playmode:
+    name: test-${{ matrix.unity_version }}-${{matrix.os}}-Playmode
+    runs-on: ${{matrix.os}}
+    if: contains(needs.check_and_prepare.outputs.platform, 'Playmode') && !cancelled()
+    needs: check_and_prepare
+    strategy:
+      fail-fast: false
+      matrix:
+        unity_version: ${{ fromJson(needs.check_and_prepare.outputs.matrix_unity_versions) }}
+        os: ${{ fromJson(needs.check_and_prepare.outputs.matrix_build_os) }}
+    env:
+      # LC_ALL, LANG and U3D_PASSWORD are needed for U3D.
+      LC_ALL: en_US.UTF-8
+      LANG: en_US.UTF-8
+      U3D_PASSWORD: ""
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Unity installer (U3D)
+        timeout-minutes: 10
+        shell: bash
+        run: |
+          gem install u3d
```

Comment:
do they fix the u3d bugs? We have to ping to an older version to get installation correct

Reply:
I think so: https://github.com/DragonBox/u3d/issues/421

Reply:
Hmm we tested 1.3.1 and it still fails, that's the reason we currently pinged to 1.2.3
https://github.com/firebase/firebase-unity-sdk/pull/300

Reply:
But the tests was running successfully: https://github.com/firebase/firebase-unity-sdk/runs/6516557068?check_suite_focus=true#step:6:1
Probably because playmode only requires the basic unity installation.